Fix 1492

ismailsunni opened the pull request:
Fix for #1492

ismailsunni commented:
Don't merge yet. It still has failed unit test.

akbargumbira commented:
is other changes necessary to fix this? seems like we just need to move plugin.py to solve

ismailsunni commented:
@akbargumbira 
I added unit test for keywordless also. But, the load_layer method can't load it. So, I update it.

I updated the Makefile, (only removing unused part).

And, I add a handler for CSV layer. See the parseurl part.
